### DISCUSSION TITLE
How to disable bulk move on same page

### DISCUSSION THREAD

**Discussion by girase1pranay** [jrief/django-admin-sortable2#409] [Q&A] [answered]
In my application I want to disable bulk move functionality, is there any configuration for that or any workaround?

**Comment by fsbraun** [comment, marked as answer]
If you mean the bulk actions (the ones in the drop down menu at the top of your changelist): You can subclass `get_actions` in your admin (the one that has the `SortableAdminMixin `), call `super()` and remove 
```
move_to_first_page
move_to_back_page
move_to_forward_page
move_to_last_page
move_to_exact_page
```

**Reply by girase1pranay** [reply]
Thanks for the answer @fsbraun.
